GameBoard/Models/GameEvent/RemoveFromGameEventViewModel.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace GameBoard.Models.GameEvent
+{
+    public class RemoveFromGameEventViewModel
+    {
+        public int GameEventId { get; set; }
+
+        public string GameEventName { get; set; }
```

Comment:
`Id` and `Name` are enough, the `GameEvent` is implied.

Reply:
Fixed.